src/map/options.html.in

```diff
@@ -169,17 +169,15 @@
                <div class="collapse chsp-map-extra-options" id="map-size-extra" style="grid-template-columns: 1fr 1fr">
                   <div class="input-group input-group-sm">
                      <label class="input-group-text" for="map-size-transform">transform:</label>
-                     <select id="map-size-transform" class="form-select" disabled>
-                        <option value="linear">linear</option>
-                        <option value="log">log</option>
-                        <option value="sqrt">sqrt</option>
-                        <option value="inverse">inverse</option>
+                     <select id="map-size-transform" class="form-select">
+                        <option value="linear" title="Linear mapping over the full range of values (large value -> large size)">linear</option>
+                        <option value="flip-linear" title="Linear reverse mapping over the full range of values (large value -> small size)">flipped linear</option>
+                        <option value="proportional" title="Proportional to actual value, breaks for negative values">proportional</option>
+                        <option value="log" title="Log-scale of actual value, breaks for negative values">log</option>
```

Comment:
The title does not show for me anywhere. Can you see it in the rendered HTML?

Reply:
the title should (and does, for me) show up as a tooltip if you hover the mouse over the option.

Reply:
Ah, right! It just take a second for it to appear